releasenotes/notes/0.11/barbell-graph-generator-5cc9df7148b1a09e.yaml

```diff
@@ -0,0 +1,14 @@
+---
+features:
+  - |
+    Added a new generator function, :func:`~retworkx.generators.barbell_graph`,
+    to the :mod:`retworkx.generators` module that will generate a barbell graph.
```

Comment:
Yeah that's expected, since it doesn't actually define it's own docstring (it's a rust python module). We just use it as a grouping namespace for the graph generator functions. I'll change this to ``retworkx.generators`` to make it clear (it should render the same way):

```suggestion
    Added a new generator function, :func:`~retworkx.generators.barbell_graph`,
    to the ``retworkx.generators`` module that will generate a barbell graph.
```